data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -63,6 +63,10 @@
 # Column subsets
 CENSUS_COUNTIES_COLUMNS = ["USPS", "GEOID", "NAME"]
 
+# Drop USVI and Guam flag for tiles
+DROP_GUAM_AND_USVI_FROM_TILES = True
```

Comment:
I'm not sure we need this bool. I like and think that `DROP_FIPS_CODE` is enough to know we're dropping some FIPS. Then on `score_post`, instead of checking the bool, you can check the length of the list and if it's greater than 0, go into the if block.

Reply:
That's also a good idea -- I could go either way.

Reply:
I like that!